extensions/resource-deployment/src/ui/resourceTypePickerDialog.ts

```diff
@@ -196,35 +195,25 @@ export class ResourceTypePickerDialog extends DialogBase {
 	}
 
 	private updateToolsDisplayTable(): void {
-		this.toolRefreshTimestamp = new Date().getTime();
-		const currentRefreshTimestamp = this.toolRefreshTimestamp;
```

Comment:
I added this to avoid the information being displayed when the selection has changed, please don't remove the related code. and looks like you are trying to improve the response time of the table?

Reply:
fixed